Nowe_Dane.md

```diff
@@ -0,0 +1,29 @@
+### Nowe Dane
+
```

Comment:
Zrealizował Pan poszukiwania w bardzo okrojonym zakresie. Zadanie miało charakter otwarty i trzeba rozszerzyć poszukiwanie. Może są jakieś portale, które
- mają dane o podobnych aktorach - 
- https://rapidapi.com/blog/list-of-online-movie-databases/ 
- co z wikipedią - tam jest pełno danych o aktorach

Reply:
Dodałem jeszcze nowe zbiory. Może wydawać się, że poszukiwanie danych było wykonane w mniejszym zakresie niż miało to miejsce w istocie. Wiele zbiorów danych jest w mojej ocenie mało przydatnych.